internal/client/v1/client/agent/rest/client.go

```diff
@@ -49,6 +51,7 @@ func New(ctx context.Context, opts ...Option) Client {
 func (c *agentClient) Exists(
 	ctx context.Context,
 	req *client.ObjectID,
+	opts ...grpc.CallOption,
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
Why the `grpc.CallOption` is needed for REST client? Is it to make a same interface with the gRPC one?

Reply:
Yes, this is the Client Interface compatibility implementation for internal/client/v1/client.
Therefore, code such as REST that does not expect gRPC Option is required on the I/F, but can be ignored by the caller due to its variable dynamic length arguments.

Reply:
I see. thanks :+1: